shell/components/form/FileSelector.vue

```diff
@@ -149,6 +149,7 @@ export default {
     v-if="!isView"
     :disabled="disabled"
     type="button"
+    role="button"
```

Comment:
Wouldn't it be wise to add `aria-label` as well? I am thinking about the visually impaired, but tbh I don't know what does the browser interpret in regards to the `<span>{{ label }}</span>` a bit further down 🤔

Reply:
I agree, it definitely makes sense to add `aria-label` to this button. The mix of the `span` along with the `input` complicates things, so it's best to be explicit in this case.